runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -363,21 +363,88 @@ class WorkItemListPage {
   }
 
   /* Workitem filter pulldown */
-  get workItemFilterPulldown () {
-//    return element(by.css(".dropdown.filter-dropdown"));
-    return element(by.css(".filter-option.pull-left"));
+  get workItemFilterFieldsPulldown () {
+    return element(by.css(".filter-fields.dropdown-toggle"));
   }
-  clickWorkItemFilterPulldown () {
-    return this.workItemFilterPulldown.click();
+  clickWorkItemFilterFieldsPulldown () {
+    return this.workItemFilterFieldsPulldown.click();
   }
+
+  get filterByAssignee () {
+    browser.wait(until.presenceOf(element(by.xpath("//li[2]/a[@class='filter-field dropdown-item']"))), constants.WAIT, 'Failed to filter by Assignee Type');
```

Comment:
We should try to replace fragile/brittle statements such as these by adding id fields to associated source file (in this case - the "id=" clause should be added to: ./runtime/node_modules/ngx-widgets/src/app/filters/filter-fields.component.html - we still may have to look for the actual string for the pulldown option, but it would be make the test code more robust and easier to maintain if we could remove as many xpath statements as possible). Thx!